optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/employee/EmployeeRestService.java

```diff
@@ -40,6 +40,14 @@
     @POST
     @Path("/add")
     Long addEmployee(@PathParam("tenantId") Integer tenantId, Employee employee);
+    
+    /**
+     * @param employee never null
+     * @return never null, the id
```

Comment:
it returns a boolean, not an id

the removeEmployee() method has the same bug, please fix that javadoc.
Let's also see if we need to return anything, boolean or Boolean.